src/Sulu/Bundle/AdminBundle/Resources/js/containers/Toolbar/Toolbar.js

```diff
@@ -20,25 +18,25 @@ const ToolbarItemTypes = {
     Dropdown: 'dropdown',
 };
 
-function getItemComponentByType(itemConfig) {
+function getItemComponentByType(itemConfig, key) {
     let item;
 
     switch (itemConfig.type) {
         case ToolbarItemTypes.Select:
-            item = (<Select {...itemConfig} />);
+            item = (<Toolbar.Select {...itemConfig} key={key} />);
             break;
         case ToolbarItemTypes.Dropdown:
-            item = (<Dropdown {...itemConfig} />);
+            item = (<Toolbar.Dropdown {...itemConfig} key={key} />);
             break;
         default:
-            item = (<Button {...itemConfig} />);
+            item = (<Toolbar.Button {...itemConfig} key={key} />);
     }
 
     return item;
 }
 
 @observer
-export default class Toolbar extends React.PureComponent<*> {
+export default class ToolbarContainer extends React.PureComponent<*> {
```

Comment:
At least until we haven't used suffixes for any component.

Reply:
its only for this file everywhere else the toolbar can be used with `Toolbar` see the `index.js` file in this directory.

Reply:
Then it is even more confusing IMO. Until now the file was named after the class, and the way it is now you would see a `ToolbarContainer` component in your react dev tools, but there wouldn't be a file named like this. Could probably end in a hard time finding stuff.